todo-app/src/task.css

```diff
@@ -0,0 +1,17 @@
+.task-list{
```

Comment:
Some minor formatting corrections would be nice, mostly in `App.js` and here in `task.css` JSX is customarily indented like HTML would be, and with CSS we usually add spaces after the colon to make it a little easier to read.